README.md

```diff
@@ -48,69 +98,375 @@ These bindings allow Azure Functions to integrate with [Azure SignalR Service](h
+{
+    return signalRMessages.AddAsync(
+        new SignalRMessage
+        {
+            // the message will be sent to the group with this name
+            GroupName = "myGroup",
+            Target = "newMessage",
+            Arguments = new [] { message }
+        });
+}
+```
+
+### 2.x C# group management output examples
+
+SignalR Service allows users to be added to groups. Messages can then be sent to a group. You can use the `SignalRGroupAction` class with the `SignalR` output binding to manage a user's group membership.
+
+#### Add user to a group
+
+The following example adds a user to a group.
+
+```csharp
+[FunctionName("addToGroup")]
+public static Task AddToGroup(
+    [HttpTrigger(AuthorizationLevel.Anonymous, "post")]HttpRequest req,
+    ClaimsPrincipal claimsPrincipal,
+    [SignalR(HubName = "chat")]
+        IAsyncCollector<SignalRGroupAction> signalRGroupActions)
+{
+    var userIdClaim = claimsPrincipal.FindFirst(ClaimTypes.NameIdentifier);
+    return signalRGroupActions.AddAsync(
+        new SignalRGroupAction
+        {
+            UserId = userIdClaim.Value,
+            GroupName = "myGroup",
+            Action = GroupAction.Add
+        });
+}
+```
+
+#### Remove user from a group
+
+The following example removes a user from a group.
+
+```csharp
+[FunctionName("removeFromGroup")]
+public static Task RemoveFromGroup(
+    [HttpTrigger(AuthorizationLevel.Anonymous, "post")]HttpRequest req,
+    ClaimsPrincipal claimsPrincipal,
+    [SignalR(HubName = "chat")]
+        IAsyncCollector<SignalRGroupAction> signalRGroupActions)
+{
+    var userIdClaim = claimsPrincipal.FindFirst(ClaimTypes.NameIdentifier);
+    return signalRGroupActions.AddAsync(
+        new SignalRGroupAction
+        {
+            UserId = userIdClaim.Value,
+            GroupName = "myGroup",
+            Action = GroupAction.Remove
+        });
+}
+```
+
+The full samples can be found [here](./samples/).
+
+#### Use SignalR Service Management SDK in functions
+
+[SignalR Service Management SDK](https://www.nuget.org/packages/Microsoft.Azure.SignalR.Management) provides communication capabilities with ASP.NET Core SignalR clients through Azure SignalR Service directly. The capabilities include sending messages to all/clients/users/groups and managing group membership. We expose the `IServicemanager` and `IServiceHubContext` in `StaticServiceHubContextStore`. For how to use Management SDK, please refer to this [guide](https://github.com/Azure/azure-signalr/blob/dev/docs/management-sdk-guide.md).
+
+SignalR input and output bindings are designed to improve user experience for Azure SignalR Service in Azure Functions. However SignalR binding is only an extrension to Azure Functions, it has some limitations, for example, you don't have an easy way to add claims into access token in `SignalRConnectionInfo`. If you have such advanced requirements that SignalR input/output binding doesn't meet, consider using `StaticServiceHubContextStore`.
```

Comment:
>to add claims into access token in `SignalRConnectionInfo`. [](start = 251, length = 59)

The description refers to Management SDK target to solve claim limitation but actually I think the advantage for Management SDK is more friendly to .NET users. Maybe we can focus here to introduce it. It doesn't solve the problem for none .NET developers anyway.

Reply:
It is inside #### 2.x C# send message output examples, other language is in another section, I think it is appropriate here